interface/sc-web-extensions/ui_components/plain-text/sc-web/components/plain_text/src/plain_text-component.js

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
Good simplification, thanks.
Just one detail: we also need to init the PaintPanel object and update the sandbox after that:

```javascript
    var paintPanel = new PlainText.PaintPanel(sandbox.container);
    paintPanel.init();
    sandbox.updateContent();
    return paintPanel;
```

Reply:
Just forgot to mention. Instead of this code:
```
    paintPanel.init();
    sandbox.updateContent();
    return paintPanel;
```
You can just add `init` call in `plain_text-paint-panel.js`:
```
PlainText.PaintPanel = function (containerId) {
	this.containerId = containerId;
	this.init();
};
```